From 63a79a81097717f6bbb5e362aaf2839c2797f480 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Sun, 18 Aug 2013 22:04:21 +0700
Subject: [PATCH 3001/3002] Remove some Japanese supports from :toolbar.menu

They are:
* Japanese dictionary editor
* Handwriting input pad

The configuration directives disable support for them, so they
actually do not work when user select them from the menu.

We use :UIM_VI_NO_CLEAN_UP to disable the items. To turn them
on, you need to define that macro at the configuration time
---
 uim-vi/src/uim-1.8.5/gtk2/toolbar/common-gtk.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/uim-vi/src/uim-1.8.5/gtk2/toolbar/common-gtk.c b/uim-vi/src/uim-1.8.5/gtk2/toolbar/common-gtk.c
index 177bb3b..2d2a97f 100644
--- a/uim-vi/src/uim-1.8.5/gtk2/toolbar/common-gtk.c
+++ b/uim-vi/src/uim-1.8.5/gtk2/toolbar/common-gtk.c
@@ -109,6 +109,7 @@ static struct _CommandEntry command_entry[] = {
     UIM_FALSE
   },
 
+#ifdef UIM_VI_NO_CLEAN_UP
   {
     N_("Japanese dictionary editor"),
     NULL,
@@ -121,6 +122,7 @@ static struct _CommandEntry command_entry[] = {
     "toolbar-show-dict-button?",
     UIM_FALSE
   },
+#endif
 
   {
     N_("Input pad"),
@@ -135,6 +137,7 @@ static struct _CommandEntry command_entry[] = {
     UIM_FALSE
   },
 
+#ifdef UIM_VI_NO_CLEAN_UP
   {
     N_("Handwriting input pad"),
     "H",
@@ -147,6 +150,7 @@ static struct _CommandEntry command_entry[] = {
     "toolbar-show-handwriting-input-pad-button?",
     UIM_FALSE
   },
+#endif
 
   {
     N_("Help"),
-- 
1.8.3

